### DISCUSSION TITLE
start_action, close_action and dpd_action

### DISCUSSION THREAD

**Discussion by keskinadem** [strongswan/strongswan#2493] [Q&A]
As I see there some changes about start_action, close_action and dpd_action in [latest versions.]([strongSwan 5.9.6](https://github.com/strongswan/strongswan/releases/tag/5.9.6)) If two device in communication but versions are different(one is 5.6.1 another one is 5.9.6), this two device can work compatible?

**Comment by tobiasbrunner** [comment]
These are changes that have only local effects (configuration, behavior when triggered).
